Corona-Warn-App/src/test/java/testhelpers/extensions/JsonExtensions.kt

```diff
@@ -0,0 +1,13 @@
+package testhelpers.extensions
+
+import com.google.gson.Gson
+import com.google.gson.JsonObject
+import okhttp3.mockwebserver.MockResponse
+
+fun String.toComparableJson() = try {
+    Gson().fromJson(this, JsonObject::class.java).toString()
+} catch (e: Exception) {
+    throw IllegalArgumentException("'$this' wasn't valid JSON")
+}
```

Comment:
how about
```suggestion
fun String.getValidatedJson(): String? = try {
    Gson().fromJson(this, JsonObject::class.java).toString()
} catch (e: Exception) {
    // log maybe
    null
}
```

Reply:
This is an extension only used in tests, I'd prefer it to fail loudly. Will keep unless you have strong feelings about this.